src/ts/playerutils.ts

```diff
@@ -59,6 +59,29 @@ export namespace PlayerUtils {
     return player.getSeekableRange() && player.getSeekableRange().start || defaultValue;
   }
 
+  /**
+   * Returns seekable time range.
+   * For live streams it will calculate timestamp timerange if tweak
+   * `enable_seek_for_live` is not present
```

Comment:
Maybe shortly describe why you need this in case this tweak isn't set. Like you described in the PR comment

Reply:
Updated https://github.com/bitmovin/bitmovin-player-ui/pull/310/commits/99ce8795e5818a4a50470324265f1b5a7e0840e4